Logger Tests Regression / Partial Reversion

### Is there an existing issue for this?

- [X] I have searched the existing issues

### SDK version

Using Lit JS SDK at commit 412264bb and beyond (master / branches after PR #579 ).

### Lit Network

Datil / or whichever network you’re using (if applicable).

### Description of the bug/issue

After the refactoring in [PR #579](https://github.com/LIT-Protocol/js-sdk/pull/579) , the `LogLevel` enum was supposed to be replaced by `LOG_LEVEL` constants from `@lit-protocol/constants`. However, subsequent merges caused a **partial rollback (ancestor revert)**:

- **Commit**: [Merge branch 'master' into staging/v7](https://github.com/LIT-Protocol/js-sdk/commit/c8801c5fe87cacee72103a4b28e0963987712b70) on Oct 3, 2024, reintroduced `LogLevel` references while also removing `export { LOG_LEVEL };` in `logger.ts`.
- This mismatch led to failing tests (TypeScript errors and incorrect timestamp comparisons), as well as confusion around whether `LogLevel` or `LOG_LEVEL` should be used.

Currently logger test fails with the error
```bash
npx nx run logger:test --coverage
:
 FAIL   logger  packages/logger/src/lib/logger.spec.ts
  ● Test suite failed to run

    packages/logger/src/lib/logger.spec.ts:1:10 - error TS2724: '"./logger"' has no exported member named 'LOG_LEVEL'. Did you mean 'LogLevel'?

    1 import { LOG_LEVEL, LogManager } from './logger';
```

We want to keep the code aligned with the overall project direction—**to unify everything under `LOG_LEVEL`**—but also maintain backwards compatibility in the short term (e.g., ensuring both `LogLevel` and `LOG_LEVEL` references don’t break the build).

### Severity of the bug

**Medium**: The logger tests fail out-of-the-box, causing confusion and blocking straightforward usage or contributions in the `logger` package.

### Steps To Reproduce

1. In this environment (Node.js v20, Nx 17.x, etc.), check out the `master` or `staging/v7` branch at or after commit `c8801c5fe87cacee7...`.  
2. Run `npx nx run logger:test --coverage` or `yarn tools --test --unit` (depending on your setup).  
3. See TypeScript errors:  
```bash
npx nx run logger:test --coverage
:
 FAIL   logger  packages/logger/src/lib/logger.spec.ts
  ● Test suite failed to run

    packages/logger/src/lib/logger.spec.ts:1:10 - error TS2724: '"./logger"' has no exported member named 'LOG_LEVEL'. Did you mean 'LogLevel'?

    1 import { LOG_LEVEL, LogManager } from './logger';
```

   Or find that tests referencing `LogLevel` fail if the code exports only `LOG_LEVEL`, etc.

### Link to code

- **Refactoring**: [PR #579](https://github.com/LIT-Protocol/js-sdk/pull/579)  
- **Commit where rollback occurred**: [c8801c5fe87cacee7...](https://github.com/LIT-Protocol/js-sdk/commit/c8801c5fe87cacee7) (Oct 3, 2024)  
- **File**: `packages/logger/src/lib/logger.ts` and `packages/logger/src/lib/logger.spec.ts` https://github.com/LIT-Protocol/js-sdk/blob/master/packages/logger/src/lib/logger.spec.ts#L1

### Anything else?

**Proposed fix**  
- Reintroduce `export { LOG_LEVEL }` in `logger.ts`, ensuring backward compatibility for any remaining references to `LogLevel`.  
- Update the tests to consistently use `LOG_LEVEL.DEBUG` (etc.) instead of `LogLevel.DEBUG`, or vice-versa if the old enum is still needed.  
- Ultimately unify everything under `LOG_LEVEL` per the original refactor’s intent.

**Context**  
- I tested a local branch that modifies `logger.spec.ts` to rely fully on `LOG_LEVEL`, while still exporting both `LOG_LEVEL` and `LogLevel` for backward compatibility.  
- After these changes, `npx nx run logger:test --coverage` passes with no TS errors and the test that checks creation timestamps is fixed to compare numeric vs. string properly.

--- a/packages/logger/src/lib/logger.spec.ts
+++ b/packages/logger/src/lib/logger.spec.ts
@@ -1,4 +1,4 @@
-import { LOG_LEVEL, LogManager } from './logger';
+import { LOG_LEVEL, LogLevel, LogManager } from './logger';
 
 describe('logger', () => {
   let lm: LogManager;
@@ -25,7 +25,7 @@ describe('logger', () => {
     lm.withConfig({
       condenseLogs: true,
     });
-    let logger = lm.get('category');
+    const logger = lm.get('category');
     expect(logger.Config?.['condenseLogs']).toEqual(true);
   });
 
@@ -33,12 +33,12 @@ describe('logger', () => {
     lm.withConfig({
       condenseLogs: true,
     });
-    let logger = lm.get('category', 'bar');
+    const logger = lm.get('category', 'bar');
     logger.setLevel(LOG_LEVEL.INFO);
     expect(logger.Config?.['condenseLogs']).toEqual(true);
     logger.info('hello');
     logger.info('hello');
-    let logs = lm.getLogsForId('bar');
+    const logs = lm.getLogsForId('bar');
     expect(logs.length).toEqual(1);
   });
 
@@ -47,7 +47,7 @@ describe('logger', () => {
     logger.setLevel(LOG_LEVEL.INFO);
     logger.info('logging');
     logger.debug('shouldnt log');
-    let logs = lm.getLogsForId('foo');
+    const logs = lm.getLogsForId('foo');
     expect(logs.length).toEqual(1);
   });
 
@@ -56,14 +56,14 @@ describe('logger', () => {
     logger.setLevel(LOG_LEVEL.DEBUG);
     logger.debug('logging');
     logger.error('error');
-    let logs = lm.getLogsForId('foo2');
+    const logs = lm.getLogsForId('foo2');
     expect(logs.length).toEqual(2);
   });
 
   it('should safe serialize circular references', () => {
     const logger = lm.get('info-logger', 'foo3');
     logger.setLevel(LOG_LEVEL.DEBUG);
-    let circ: any = { foo: 'bar' };
+    const circ: any = { foo: 'bar' };
     circ.circ = circ;
     logger.debug('circular reference to serialize', circ);
     console.log(lm.getLogsForId('foo3'));
@@ -122,7 +122,8 @@ describe('logger', () => {
     const requestIds = lm.LoggerIds;
 
     expect(requestIds.length).toBe(2);
-    expect(loggerA.timestamp).toEqual(requestIds[0]);
-    expect(loggerB.timestamp).toEqual(requestIds[1]);
+    expect(loggerA.timestamp).toBeLessThan(loggerB.timestamp);
+    expect(requestIds[0]).toBe('1');
+    expect(requestIds[1]).toBe('2');
   });
 });

--- a/packages/logger/src/lib/logger.ts
+++ b/packages/logger/src/lib/logger.ts
@@ -1,6 +1,7 @@
 import { version, LOG_LEVEL, LOG_LEVEL_VALUES } from '@lit-protocol/constants';
 import { hashMessage } from 'ethers/lib/utils';
 
+export { LOG_LEVEL };
 export enum LogLevel {
   OFF = -1,
   ERROR = 0,

